metricbeat/module/openmetrics/_meta/fields.yml

```diff
@@ -7,18 +7,41 @@
   fields:
     - name: openmetrics
       type: group
+      #      release: beta/ga
       description: >
         `openmetrics` contains metrics from endpoints that are following Openmetrics format.
       fields:
         # Order is important here, labels will match first, the rest are double
+        - name: help
+          type: keyword
+          description: >
+            Brief description of the MetricFamily
+        - name: type
+          type: keyword
+          description: >
+            Metric type
+        - name: unit
+          type: keyword
+          description: >
+            Metric unit
```

Comment:
Do we want to store description type and unit? which ones would we use when a document contains multiple metrics?

Reply:
I think it will be useful. Let me know if you would like it to be removed.

Reply:
But what values would be included in documents with multiple metrics?

Reply:
Help, Type, Unit values will be taken from scraped metrics and will be associated with the metric they refer to. 
Added examples to test files - 
  metricbeat/module/openmetrics/collector/_meta/testdata/openmetrics-features.plain
  metricbeat/module/openmetrics/collector/_meta/testdata/openmetrics-features.plain-expected.json

Reply:
Ah ok, it seems we are sending only one metric per document in this collector, is this correct?

In the prometheus collector I think that we were sending metrics with the same set of labels together in the same document, this is important for saving storage space. I thought we were going to do the same here.

@ChrsMark @exekias wdyt about including this help/type/unit values in the documents?

Reply:
1. Storing metrics of the same labels set together is sth important I think. Shall we follow the same approach here? Not sure if doable if we decide to add the extra fields too though.
2. I'm not quite sure. Do we actually see real value in those? If so we can consider it, otherwise it would be better if we keep them out for now and benefit from the storage optimisation (same set of labels stored together). Actually I had the same question about exemplars at https://github.com/elastic/beats/pull/27269#discussion_r699324963 :)

Reply:
@jsoriano , @ChrsMark , I have introduced 2 config parameters, `enable_exemplars`, `enable_metadata` to control output of Exemplars and Type/Unit/Help metadata. By default these flags will be `false`. With this approach, users who want Exemplars and Metadata can still use this feature. I hope this addresses your concerns.
Thanks.

Reply:
There is still the issue about grouping metrics with the same labels, this is something we probably want, as we do with prometheus.

And if we group multiple metrics in the same document, I don't think we should add they types metadata.

For exemplars I am not sure, but probably we want to send them in separate events.

Reply:
@jsoriano  Added test case to show grouped metrics with same labels, when metadata and exemplars are turned off.
Please check
https://github.com/elastic/beats/pull/27269/files/414a5f07a740582a9298bdc86573a5c396eaaaf9..2269f9dfff23e4b1c432835d071d18b5b78683cd#diff-e7218251866dff4b44deab13b2345bddb50de4907edcf955e92c1cef348f0226R42-R45

Reply:
Great, thanks. So now metrics are grouped by default, but if exemplars or metadata are enabled, then they are not grouped, right? This sounds like a reasonable approach to me.

Reply:
I'm ok too since the default case works with the grouped approach. Thanks @premendrasingh !